### ISSUE TITLE
Python-version denne fungerer på?

### ISSUE THREAD

**Issue by alekslyse** [stenjo/SbankenToYNAB#33]
Er det en Python-version denne fungerer på? Evt noen måte å få det til å fungere? Ptt fungerer ikke scriptet på 3.9 eller 3.8

**Comment by alekslyse** [comment]
Ja, den fungerte med 3.7, men trenger vel en overhaling en del. Hvis en legger inn et lån med rente etc feiler scriptet da ynab knytter lånet til en katagori. 

`ValueError: Invalid value for `type` (autoLoan), must be one of ['checking', 'savings', 'creditCard', 'cash', 'lineOfCredit', 'merchantAccount', 'payPal', 'investmentAccount', 'mortgage', 'otherAsset', 'otherLiability']`

**Comment by stenjo** [comment]
Ja, den siste der har jeg forsåvidt fikset i en pr jeg jobber med. Du kan jo prøve den.
https://github.com/stenjo/SbankenToYNAB/pull/32

Advarsel: Dette er work in progress, så - som alltid - ingen garantier for at dette fungerer :-)

**Comment by alekslyse** [comment]
> Ja, den siste der har jeg forsåvidt fikset i en pr jeg jobber med. Du kan jo prøve den. #32
> 
> Advarsel: Dette er work in progress, så - som alltid - ingen garantier for at dette fungerer :-)

Den var ikke stabil, med mindre den må kjøres på en spesiell måte.

`python3.7 sync_accounts.py 
Traceback (most recent call last):
  File "sync_accounts.py", line 3, in <module>
    import api_settings
ModuleNotFoundError: No module named 'api_settings'`

**Comment by stenjo** [comment]
Har du husket å kopiere `api_settings_format.py` til `api_settings.py` og lagt inn nøklene?

**Comment by alekslyse** [comment]
Takk det fungerte:

Legg til "studentLoan", "otherDebt", "personalLoan" også, så dekker du de andre lånene
